### DISCUSSION TITLE
Add 72mhz for rc submarines

### DISCUSSION THREAD

**Discussion by jorgeneo560** [pascallanger/DIY-Multiprotocol-TX-Module#711] [Ideas]
rc submarine builders strugles to find tx and rx that work on submarines, 2.4ghz it just bounce the surface of the water like a tennis ball, old 72 mhz radios are capable to give 10-15 m of control before fail safe kicks in, now on this day's theres almost imposible to find new equipment, some long range fpv tx in the freq of 433 and 900 works best but is far from the glory of 72mhz, some people are force to extend 2 or 3 m the rx antenna and put a buyance ball in the end to avoid lost of control when dive, but is not a looking scale solution either, and had to watch out from the propeller cutting the cable.
A lot of people abandon the dream of a rc submarine becouse of this limitation, who already are in the hobby buy old 2nd handed old radios to make the model work.
It would be nice if you can make a modern 72 mhz tx modules for the new radio with the telemetry option

**Comment by pascallanger** [comment]
Wilhelm has done it, contact him by email: wilhelm.wm.meier at googlemail.com

**Comment by geofrancis** [comment]
we are working on a 27mhz system for submarines if you want to help, 27mhz is a worldwide band where 72mhz is US and canada only. 
https://www.theassociationofmodelsubmariners.com/t1972-27mhz-radio-ideas#11771
https://github.com/rdforrest/27MHz-using-AX5043

**Reply by jorgeneo560** [reply]
nice project, yes i would be happy to help

**Comment by wimalopaan** [comment]
Very cool!

If have made an DDS-FM-RF-Modul based on AD9851 (vry expensive) and one using SI5351. These can be used with OpenTx/EdgeTx because they implement the MPM-Protocol (Pascal reserved a protocol number for that). Both work great with e.g. Corona 40MHz scan-receiver RP-8. One big disadvantage is the long antenna.

The actual project is to use the 2.4GHz or other link to a 2,4GHz receiver and this RX ist connected via SBus/IBus to my RF-FM-Modul. So the transmitter has only the handy short antenna and the FM-Modul stand on a tripot and uses a real lambda/4 ground plane antenna.


![IMG_20220608_175014_7 (1)](https://user-images.githubusercontent.com/8696319/180744479-ba17f1a0-9a1b-4b65-9374-1d4b71c7a065.jpg)

![IMG_20220608_175046_0 (1)](https://user-images.githubusercontent.com/8696319/180744509-9d8787cf-433a-4ffa-b6ad-6062d965e233.jpg)

**Reply by geofrancis** [reply]
can you share details or code?
